# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../stellarium/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur stellarium-0.7.1-orig/src/observator.h stellarium-0.7.1/src/observator.h
--- stellarium-0.7.1-orig/src/observator.h	2005-09-08 16:24:45.000000000 +0000
+++ stellarium-0.7.1/src/observator.h	2006-04-02 23:07:22.797708500 +0000
@@ -112,8 +112,8 @@
 	string s_time_format_to_string(S_TIME_FORMAT) const;
 
 	// Convert the date format enum to its associated string and reverse
-	S_DATE_FORMAT Observator::string_to_s_date_format(const string& df) const;
-	string Observator::s_date_format_to_string(S_DATE_FORMAT df) const;
+	S_DATE_FORMAT string_to_s_date_format(const string& df) const;
+	string s_date_format_to_string(S_DATE_FORMAT df) const;
 
 	// for changing position
 	bool flag_move_to;
diff -Nur stellarium-0.7.1-orig/src/projector.h stellarium-0.7.1/src/projector.h
--- stellarium-0.7.1-orig/src/projector.h	2005-07-03 19:54:21.000000000 +0000
+++ stellarium-0.7.1/src/projector.h	2006-04-02 23:06:59.860275000 +0000
@@ -75,7 +75,7 @@
 	virtual void set_viewport(int x, int y, int w, int h);
 
 	// Fill with black around the circle
-	void Projector::draw_viewport_shape(void);
+	void draw_viewport_shape(void);
 
 	int viewW(void) const {return vec_viewport[2];}
 	int viewH(void) const {return vec_viewport[3];}
diff -Nur stellarium-0.7.1-orig/src/stel_command_interface.h stellarium-0.7.1/src/stel_command_interface.h
--- stellarium-0.7.1-orig/src/stel_command_interface.h	2005-08-12 16:02:42.000000000 +0000
+++ stellarium-0.7.1/src/stel_command_interface.h	2006-04-02 23:08:20.957343250 +0000
@@ -43,9 +43,9 @@
  public:
   StelCommandInterface(stel_core * core);
   virtual ~StelCommandInterface();
-  virtual int StelCommandInterface::execute_command(string commandline);
-  virtual int StelCommandInterface::execute_command(string command, double arg);
-  virtual int StelCommandInterface::execute_command(string command, int arg);
+  virtual int execute_command(string commandline);
+  virtual int execute_command(string command, double arg);
+  virtual int execute_command(string command, int arg);
   virtual int execute_command(string command, unsigned long int &wait, bool trusted);
   virtual int set_flag(string name, string value, bool &newval, bool trusted);
   void update(int delta_time);
diff -Nur stellarium-0.7.1-orig/src/stel_ui.h stellarium-0.7.1/src/stel_ui.h
--- stellarium-0.7.1-orig/src/stel_ui.h	2005-09-08 16:41:12.000000000 +0000
+++ stellarium-0.7.1/src/stel_ui.h	2006-04-02 23:12:28.508814250 +0000
@@ -316,7 +316,7 @@
 	void tui_cb_sky_time();        // change effect parameters
 
 	// Parse a file of type /usr/share/zoneinfo/zone.tab
-	s_tui::MultiSet_item<string>* stel_ui::create_tree_from_time_zone_file(const string& zonetab);
+	s_tui::MultiSet_item<string>* create_tree_from_time_zone_file(const string& zonetab);
 
 	bool ScriptDirectoryRead;
 };
